FBC/DebtAndEquities/Debt.rdf

```diff
@@ -951,6 +965,13 @@
 		<skos:definition>relates something, such as an agreement, or debt instrument, to the rate (typically annual) of interest that is to be paid by the debtor to the creditor on the debt</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fbc-dae-dbt;hasInterestRateCap">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-dae-dbt;hasInterestRate"/>
+		<rdfs:label>has interest rate cap</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fbc-dae-dbt;InterestRateCap"/>
```

Comment:
It's redundant to have a specific class here - just use InterestRate.

Reply:
@rivettp See my comment above regarding the fact that this is a required field for regulatory reporting, and that it can be difficult to map across repos without a class to map to - that allows for format inconsistencies across repos for example, which use of a single property makes more difficult.